File: Semana 3/Tarea encargada de funciones (cuaderno)/C++/Ejercicio7.cpp
#include <iostream>
#include <cmath>
using namespace std;

bool Ejercicio7(int n) {
    if (n < 2) return false;
    for (int i = 2; i <= sqrt(n); i++) {
        if (n % i == 0)
            return false;
    }
    return true;
}

int main() {
    cout << "13 es primo? " << (Ejercicio7(13) ? "Si" : "No") << endl;
    cout << "15 es primo? " << (Ejercicio7(15) ? "Si" : "No") << endl;
    return 0;
}

File: Semana 3/Tarea encargada de funciones (cuaderno)/python/Ejercicio7.py
import math

def ejercicio7(n):
    if n < 2:
        return False
    for i in range(2, int(math.sqrt(n)) + 1):
        if n % i == 0:
            return False
    return True

print(f"13 es primo? {'Si' if ejercicio7(13) else 'No'}")
print(f"15 es primo? {'Si' if ejercicio7(15) else 'No'}")
